assets/constants.ts

```diff
@@ -18,7369 +18,7391 @@
  */
 
 export default {
-  // The term 'staging' is used instead of the classroom url fragment field
-  // in the URL for topics that are not yet attached to a classroom.
-  "CLASSROOM_URL_FRAGMENT_FOR_UNATTACHED_TOPICS": "staging",
+    // The term 'staging' is used instead of the classroom url fragment field
```

Comment:
Why are we changing this file? (Looks like some config issue with your IDE)

Reply:
Made the necessary changes